understanding/language-of-parts.html

```diff
@@ -58,186 +56,136 @@
-                  				SC 3.1.2:</span>Language of Parts (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.1.2:</span>一部分の言語 (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>The <a href="#dfn-human-language">human language</a> of each passage or phrase in the content can be <a href="#dfn-programmatically-determined">programmatically determined</a> except for proper names, technical terms, words of indeterminate language, and words
-                     or phrases that have become part of the vernacular of the immediately surrounding
-                     text.
+                  <p>コンテンツの一節、又は語句それぞれの<a href="#dfn-human-language">自然言語</a>がどの言語であるか、<a href="#dfn-programmatically-determined">プログラムによる解釈が可能</a>である。ただし、固有名詞、技術用語、言語が不明な語句、及びすぐ前後にあるテキストの言語の一部になっている単語又は語句は除く。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Assistive technology can identify the languages used within a page.</dd>
+                  <dt>目標</dt>
+                  <dd>支援技術は、ページ内で使用されている言語を特定できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Indicate when words are in a different language.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>単語が異なる言語で書かれていることを示す。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People using assistive technology get information in the correct language.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>支援技術を使用するユーザーは、正しい言語で情報を取得できる。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that user agents can correctly present
-                  phrases, passages, and in some cases words written in multiple languages. This makes
-                  it possible for user agents and
-                  assistive technologies to present content according to the  presentation and pronunciation
-                  rules for that language. This applies to graphical browsers as well as screen readers,
-                  braille displays, and other voice browsers.
+               <h2>意図</h2>
+               <p>この達成基準の目的は、ユーザエージェントが複数の言語で書かれた語句、引用部分、場合によっては単語を正しく提示できることを保証することである。これにより、その言語の提示規則および発音規則に従ってコンテンツをユーザエージェントおよび支援技術に提示することが可能になる。これは、スクリーンリーダー、点字ディスプレイ及びその他の音声ブラウザだけでなくグラフィカルブラウザも適用される。
```

Comment:
phrases を「**語**句」と訳すと、後に出てくる「words (語)」と一部被ってしまう印象を受けたので、「句」とするのはいかがでしょうか。また、passage は「節」でいかがでしょうか。

```suggestion
               <p>この達成基準の目的は、ユーザエージェントが複数の言語で書かれた句、節、場合によっては単語を正しく提示できることを保証することである。これにより、その言語の提示規則および発音規則に従ってコンテンツをユーザエージェントおよび支援技術に提示することが可能になる。これは、スクリーンリーダー、点字ディスプレイ及びその他の音声ブラウザだけでなくグラフィカルブラウザも適用される。
```

Reply:
「語句」を「句」に修正しました。
「引用部分」は英和辞書にありますので、そのままとさせてください。